diagnostics-app/src/app/telemetry/telemetry-card/telemetry-card.component.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Defines the Card component for telemetry.
+ * Imported by telemetry.module.ts
+ */
+
+import { Component, Input } from '@angular/core';
+import { TelemetryInfoType } from '@common/message';
+
+@Component({
+  selector: 'app-telemetry-card',
+  templateUrl: './telemetry-card.component.html',
+  styleUrls: ['./telemetry-card.component.css'],
+})
+export class TelemetryCardComponent {
+  @Input() type: TelemetryInfoType = TelemetryInfoType.DEFAULT;
```

Comment:
Could we use `undefined` for enum?

Reply:
The value of 'type' will be undefined for the very first moment, so the template will check if 'type' is equal to undefined, it will not render the content. The input is also set to be required.